frontend/chat-plugin/handles/index.ts

```diff
@@ -1,3 +1,4 @@
 export const cyBubble = 'bubble';
 export const cyInputbarTextarea = 'inputbarTextarea';
 export const cyInputbarButton = 'inputbarButton';
+export const cyChatPluginListChat = 'chatPluginListChat';
```

Comment:
naming is hard but important we should make clear that this is the message list. Maybe chatPluginMessageList ?